include/client/graphics/Player.h

```diff
@@ -49,6 +53,7 @@ class Player : public GameThing, InputListener {
     if (fx_land) delete fx_land;
     if (fx_item) delete fx_item;
     if (fx_tag) delete fx_tag;
+    if (sfx_jump) delete sfx_jump;
```

Comment:
can you delete the other sfx resources too?

Reply:
added in newest commit